go.mod

```diff
@@ -59,7 +64,7 @@ require (
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
 	github.com/mvdan/xurls v1.1.0
 	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/gomega v1.10.3
```

Comment:
Why all these other version bumps?

Reply:
@jpbetz Apologies, actually I stopped working on it because the changes were not correct as suggested by pcoxu. I have made https://github.com/kubernetes/kubernetes/pull/10722 instead in which etcd dep are upgraded.